### DISCUSSION TITLE
Cara membuat tabel persentase tapi per row

### DISCUSSION THREAD

**Discussion by edbert2397** [teamalgoritma/community#57] [Q&A] [answered]
Halo semuanya, saya ingin bertanya
misal saya memiliki table seperti ini
2 3
5 10
saya ingin membuat table persentase tapi hanya untuk sesama row,menjadi seperti ini misal
40% 60%
33% 66%
mohon bantuannya yaa

**Comment by inytss** [comment, marked as answer]
Dear @edbert2397, ingin mengonfirmasi ulang apakah yang Bapak maksud adalah dalam 1 row jika dijumlahkan persentasenya sama dengan 100%? 
Jika iya, maka Bapak dapat mencoba menggunakan `prop_table(table(), margin = 1)`. Penggunaan `margin=1` artinya kita ingin membuat table proporsi yang mana dia akan menghitung proporsinya per baris datanya.

**Reply by edbert2397** [reply]
wah iya benar,terima kasih ya kak
